marketplace/config.json

```
@@ -1,12 +1,470 @@
+    "site": "https://stakestone.io"
+  },
+  {
+    "id": "relend-network",
+    "external": true,
+    "title": "Relend Network",
+    "logo": "/assets/relend.svg",
+    "shortDescription": "Relend Network is a decentralized lending protocol focused on efficient asset lending and borrowing.",
+    "categories": ["Lending"],
+    "author": "Relend Network",
+    "url": "https://relend.network",
+    "description": "Relend Network streamlines lending processes and offers competitive borrowing rates across decentralized platforms.",
+    "site": "https://relend.network"
+  },
+  {
+    "id": "mint-park",
+    "external": true,
+    "title": "Mint Park",
+    "logo": "/assets/mintpark.svg",
+    "shortDescription": "Mint Park is an NFT marketplace built for Bitcoin Layer 2 networks.",
+    "categories": ["NFTs"],
+    "author": "Mint Park",
+    "url": "https://mintpark.io",
+    "description": "Mint Park provides an NFT marketplace optimized for Bitcoin Layer 2 networks, enabling seamless NFT creation and trading.",
+    "site": "https://mintpark.io"
+  },
+  {
+    "id": "hemi-bros",
+    "external": true,
+    "title": "Hemi Bros",
+    "logo": "/assets/hemibros.jpeg",
+    "shortDescription": "Hemi Bros is an NFT platform for creating and trading unique digital collectibles.",
+    "categories": ["NFTs"],
+    "author": "Hemi Bros",
+    "url": "https://hemibros.example",
+    "description": "Hemi Bros offers a platform for artists and collectors to create, showcase, and trade NFTs.",
+    "site": "https://hemibros.example"
+  },
+  {
+    "id": "metamask",
+    "external": true,
+    "title": "Metamask",
+    "logo": "https://images.ctfassets.net/clixtyxoaeas/1ezuBGezqfIeifWdVtwU4c/d970d4cdf13b163efddddd5709164d2e/MetaMask-icon-Fox.svg",
+    "shortDescription": "Metamask is a widely used wallet for interacting with Ethereum and other EVM chains.",
+    "categories": ["Wallets"],
+    "author": "Metamask",
+    "url": "https://metamask.io",
+    "description": "Metamask enables users to manage digital assets and interact with decentralized applications through a secure browser extension and mobile wallet.",
+    "site": "https://metamask.io"
+  },
+  {
+    "id": "pure-finance",
+    "external": true,
+    "title": "Pure Finance",
+    "logo": "/assets/pure-finance.png",
+    "shortDescription": "Pure Finance is a set of EVM tools and smart contracts offering solutions for developers, DeFi applications, and individuals alike.",
+    "categories": ["Dev tooling"],
+    "author": "Pure Finance",
+    "url": "https://purefinance.hemi.xyz/",
+    "description": "Pure Finance is a set of EVM tools and smart contracts offering solutions for developers, DeFi applications, and individuals alike.",
+    "site": "https://purefinance.hemi.xyz/"
+  },
+  {
+    "id": "demos",
+    "external": true,
+    "title": "Demos",
+    "logo": "",
```

Do we want to remove this logo? Perhaps it should link to the local asset:

```suggestion
    "logo": "https://raw.githubusercontent.com/hemilabs/blockscout-config/refs/heads/master/assets/demos.png",
```